test/unit/tile.js

```diff
@@ -146,13 +146,38 @@ describe('Tile', function () {
 
   it('Prevent larger overlap than default size', function () {
     assert.throws(function () {
-      sharp().tile({overlap: 257});
+      sharp().tile({
```

Comment:
I notice there are code layout changes unrelated to this PR. Is this using `prettier`? (Not a problem, more for interest.)

Reply:
I use atom editor and this is the result of Beautify. I like it, but If you prefer to be consistent with the other convention you were using I can revert these changes.